core/src/main/java/io/questdb/cutlass/http/processors/JsonQueryProcessor.java

```diff
@@ -331,26 +331,34 @@ private static void logInternalError(
         if (e instanceof CairoException) {
             CairoException ce = (CairoException) e;
             if (ce.isInterruption()) {
-                state.info().$("query cancelled [q=`").utf8(state.getQuery()).$("`, reason=`").$(((CairoException) e).getFlyweightMessage()).$("`]").$();
+                state.info().$("query cancelled [q=`").utf8(state.getQuery())
+                        .$("`, reason=`").$(((CairoException) e).getFlyweightMessage()).$("`]").$();
             } else {
                 if (ce.isCritical()) {
-                    state.critical().$("error [q=`").utf8(state.getQuery()).$(", msg=`").$(ce.getFlyweightMessage()).$('`').$(", errno=`").$(ce.getErrno()).I$();
+                    state.critical().$("error [q=`").utf8(state.getQuery())
+                            .$(", msg=`").$(ce.getFlyweightMessage()).$('`')
+                            .$(", errno=`").$(ce.getErrno()).I$();
                 } else {
-                    state.error().$("error [q=`").utf8(state.getQuery()).$(", msg=`").$(ce.getFlyweightMessage()).$('`').$(", errno=`").$(ce.getErrno()).I$();
+                    state.error().$("error [q=`").utf8(state.getQuery())
+                            .$(", msg=`").$(ce.getFlyweightMessage()).$('`')
+                            .$(", errno=`").$(ce.getErrno()).I$();
                 }
             }
         } else if (e instanceof HttpException) {
```

Comment:
Doesn't it remove the ` at the end of logged message ?

Reply:
There is `.$('`')` in the `.$(", msg=`").$(ce.getFlyweightMessage()).$('`')` call chain. Are you referring to something else? This change is nothing but reformating.

Reply:
I meant ` around reason in  : 

```
state.error().$("internal HTTP server error [q=`").utf8(state.getQuery()).$("`, reason=`").$(((HttpException) e).getFlyweightMessage()).$("`]").$();

```
vs
```
state.error().$("internal HTTP server error [q=`").utf8(state.getQuery())
                    .$("`, reason=`").$(((HttpException) e).getFlyweightMessage()).I$();
```

Reply:
I see. Improved the formatting in e89b566cf224835a8df3f046bcaaa69289ab407f